src/Command/CodeStudio/CodeStudioDisableDefaultJobCommand.php

```diff
@@ -0,0 +1,281 @@
+<?php
+
+namespace Acquia\Cli\Command\CodeStudio;
+
+use Acquia\Cli\Command\CommandBase;
+use Acquia\Cli\Exception\AcquiaCliException;
+use Acquia\DrupalEnvironmentDetector\AcquiaDrupalEnvironmentDetector;
+use Gitlab\Client;
+use Gitlab\Exception\RuntimeException;
+use Gitlab\HttpClient\Builder;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+
+/**
+ * Class CodeStudioDisableDefaultJobCommand.
+ */
+class CodeStudioDisableDefaultJobCommand extends CommandBase {
+
+  /**
+   * @var string
+   */
+  private $gitlabHost;
+
+  /**
+   * @var string
+   */
+  private $gitlabToken;
+
+  /**
+   * @var \Gitlab\Client
+   */
+  private $gitLabClient;
+
+  /**
+   * @var array
+   */
+  private $gitLabAccount;
+
+  protected static $defaultName = 'codestudio:disable-default-job';
+
+  /**
+   * {inheritdoc}.
+   */
+  protected function configure() {
+    $this->setDescription('Disable default jobs provided by the Code Studio.')
```

Comment:
command name is updated to `codestudio:set-cicd-var` so that now any variable can be updated (from supported list). Supported list of variables can be checked with `--help` option